src/lib/index.ts

```diff
@@ -1,7 +1,38 @@
 import fetch from 'node-fetch';
 import path from 'path';
 import execSh from 'exec-sh';
-import { specmaticJarPathLocal } from '../config';
+import { specmaticJarPathLocal, specmatic } from '../config';
+import fs from 'fs';
+import { check } from 'yargs';
+
+type Environment = Record<string, string>
+
+export const setSpecmaticEnvironement = (environmentName: string, environment: Environment) => {
+  console.log("Reading the file from -> ", path.resolve(specmatic))
+  let file = null
+
+  try {
+    file = require(path.resolve(specmatic))
+    for (var x in environment) { file.environments[environmentName].variables[x] = environment[x] }
```

Comment:
1) use a better variableName, `environmentVariable` 
2) there is no need for {} when there is only one statement inside the block

Reply:
Made the following changes